src/App/Repository/Post/TagsRepository.php

```diff
@@ -88,16 +110,18 @@ public function updateValue($id, $value)
     protected function parseTag($tag)
     {
         if (is_array($tag)) {
-            $tag = $tag['id'];
+            return $tag['id'];
+        }
+
+        if (is_numeric($tag)) {
```

Comment:
This makes a new assumptions that a numeric tag is always the ID. I did consider dropping this parsing entirely, but I think we might be relying on it for CSV imports to map tag names in the CSV to an ID.

Reply:
We are definitely relying on it so it needs to remain sadly, it's actually hugely useful in relation to tagging.

Reply:
Do you think it needed to remain as it was? or is this partial refactor (assuming numeric = tag id) ok?

Reply:
For the purposes of the importer I could pass tags as `['id' => 1]` style and that'd be completely safe to skip the lookup on

Reply:
No no I think this is fine